### PR TITLE
[Move] td-uefi-pi is migrating main branch

### PR DESCRIPTION
**Pull request by haowqs**
Migrate the staging branch uefi-pi to the main branch td-uefi-pi

Signed-off-by: haowx <weix.hao.intel.com>

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -10,7 +10,7 @@ members = [
 	"td-layout",
 	"td-logger",
 	"td-paging",
-	"td-uefi-pi",
```
Comment (jiangliu) [inline Cargo.toml:13]: It would be better to keep it as `td-uefi-pi`, so we have a common prefix for all crates

**Comment by jiangliu** [comment]
The crate is specially designed for td, so it would be better to keep it as `td-uefi-pi`.
And the r-uefi-pi should already have been merged into td-uefi-pi, so just need to merge several PRs from staging into main.

File: Cargo.toml (hunk 1 above)
Comment (haowqs) [reply Cargo.toml]: Done

**Comment by haowqs** [comment]
I changed the commit info.

**Review by jyao1 — APPROVED** [review]

**Review by jiangliu — APPROVED** [review]

**Pull request merged**